pages/profiles/[publicKey]/nfts.tsx

```diff
@@ -71,7 +71,7 @@ export const NFTCard = ({
   loading = false,
 }: {
   nft: OwnedNFT;
-  marketplace: Marketplace;
+  marketplace: {auctionHouse: AuctionHouse};
```

Comment:
I see the intent since auction house is the only thing used, has this been verified to work on profiles and nft details as well cause i think the initial reason for including the whole Marketplace was that there might be a need in the future for stats and other marketplace fields for the forms as it matures?

Reply:
I've updated everything based on what's used in the various forms, and using typescript type checking, so it _should_ be safe. 

@kristianeboe and I both agree, it's best to have a minimal interface that requests only the needed stuff, and then you can pass in the whole object (`Marketplace`) in this case, which makes future expansion/refactoring simpler. This also avoids some graph-ql antipatterns, especially requesting more data than are needed.